src/kameleon/util/search.clj

```diff
@@ -3,7 +3,7 @@
 
 (defn format-query-wildcards
   [search-term]
-  (str/replace search-term #"[%_*?]" {"%" "\\%",
+  (str/replace (str/trim search-term) #"[%_*?]" {"%" "\\%",
                                       "_" "\\_",
                                       "*" "%",
                                       "?" "_"}))
```

Comment:
This looks like the right fix to me.

It could use a little code-style cleanup.
The 3rd parameter to `str/replace` (everything between `{` and `}`) is a map, and its key-value pairs are each on their own line. These key-values pairs were all lined up below the 1st pair before this change. So you could add some more leading space to the following lines to get them to line up again. This will help keep this code more readable in the future:

```suggestion
  (str/replace (str/trim search-term) #"[%_*?]" {"%" "\\%",
                                                 "_" "\\_",
                                                 "*" "%",
                                                 "?" "_"}))
```

Reply:
thank you! That was informative and clears up some of my questions.

Reply:
Great! No problem 👍